.github/ISSUE_TEMPLATE/electionquestion.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Election Question
+about: Do you have a question for the Board of Director election candidates? Submit it here!
+---
```

Comment:
Didn't we say in the chat there would be an automatic label assigned here

```
labels: candidate-question
```

Reply:
@ahoefling awesome, I missed that capability. Thanks for the catch! Will add it.

Reply:
team work makes the dream work. Everything else looks great, good job!